### PR TITLE
pkp/pkp-lib#278 Forward-port permissions improvements to OJS 3.0/OMP 1.2

### PR DESCRIPTION
**Pull request by asmecher**
For pkp/pkp-lib#278

### CODE REVIEW

File: classes/submission/Submission.inc.php [hunk 1]
```diff
@@ -1120,6 +1246,32 @@ function setDatePublished($datePublished) {
 		return $this->SetData('datePublished', $datePublished);
 	}
 
+	/**
+	 * Initialize the copyright and license metadata for an article.
```
Comment (jnugent) [inline classes/submission/Submission.inc.php]: Minor thing, but maybe generalize 'article' to 'submission' for pkp-lib

File: classes/submission/Submission.inc.php [hunk 2]
```diff
@@ -1120,6 +1246,32 @@ function setDatePublished($datePublished) {
...
+	function initializePermissions() {
+		$this->setLicenseURL($this->getDefaultLicenseURL());
+		$this->setCopyrightHolder($this->getDefaultCopyrightHolder(null), null);
+		if ($this->getStatus() == STATUS_PUBLISHED) {
+			$this->setCopyrightYear($this->getDefaultCopyrightYear());
+		}
+	}
+
+	/**
+	 * Determines whether or not the license for copyright on this Article is
```
Comment (jnugent) [inline classes/submission/Submission.inc.php]: Same.

Comment (asmecher) [inline classes/submission/Submission.inc.php]: Done!

File: classes/submission/Submission.inc.php (hunk 1 above)
Comment (asmecher) [inline classes/submission/Submission.inc.php]: Done!

**Pull request merged**